src/features/user/Widget/DonationLink/DonationLinkForm.tsx

```diff
@@ -0,0 +1,244 @@
+              {t('donationLink:projectTitle')}
+            </div>
+            <ProjectSelectAutocomplete
+              handleProjectChange={handleProjectChange}
+              project={localProject}
+              projectList={projectsList || []}
+              active={true}
+            />
+          </div>
+          <InlineFormDisplayGroup>
+            <div className={styles.formHeader}>
+              {t('donationLink:treeCounterTitle')}
+            </div>
+            <Switch
+              checked={isSupport}
+              onChange={() => {
+                setIsSupport(!isSupport);
+              }}
+              disabled={user.isPrivate}
+            />
+          </InlineFormDisplayGroup>
+          <InlineFormDisplayGroup>
+            <div className={styles.formHeader}>
+              {t('donationLink:testingTitle')}
+            </div>
+            <Switch
+              checked={isTesting}
+              onChange={() => {
+                setIsTesting(!isTesting);
+              }}
+              disabled={false}
+            />
+          </InlineFormDisplayGroup>
+          <div className={styles.singleFormGroup}>
+            <div className={styles.formHeader}>
+              {t('donationLink:urlTitle')}
+            </div>
+            <InlineFormDisplayGroup>
+              <TextField
+                id="outlined-read-only-input"
+                InputProps={{
+                  readOnly: true,
+                }}
+                value={donationUrl}
+                onChange={handleChange}
+              />
+              <CopyToClipboard isButton={true} text={donationUrl} />
+            </InlineFormDisplayGroup>
+          </div>
+          <div className={styles.formButtonContainer}>
+            <Button
+              id={'Preview'}
+              variant="contained"
+              color="primary"
+              fullWidth={false}
+              onClick={() => window.open(donationUrl, '_blank')}
+              disabled={isProjectSelected}
+            >
+              {t('donationLink:preview')}
+            </Button>
+            {/* <Button
+              id={'Copy'}
+              variant="outlined"
+              color="primary"
+              fullWidth={false}
+              onClick={() => setTextCopiedClipboard()}
+              disabled={isProjectSelected}
+            >
+              {t('donationLink:copy')}
+            </Button>
+            <Snackbar
+              open={isCopied}
+              autoHideDuration={4000}
+              onClose={setTextCopiedClipboard}
+              anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+            >
+              <div>
+                <Alert
+                  elevation={6}
+                  variant="filled"
+                   onClose={setTextCopiedClipboard}
+                  severity="success"
+                >
+                  {t('donate:Copied To Clipboard')}
+                </Alert>
+              </div>
+            </Snackbar> */}
```

Comment:
Remove commented code if no longer needed. You can refer to the code in an earlier version.